internal/armada/server/lease.go

```diff
@@ -16,6 +16,8 @@ import (
 	"github.com/G-Research/armada/pkg/api"
 )
 
+const maxRetries = 5
```

Comment:
Should this be part of the scheduling config?

Reply:
yes